README.md

```
@@ -1,212 +1,110 @@
+6. **Missie & Visie** - Gezamenlijke samenvatting op basis van alle antwoorden
 
-### Development Guidelines
+## API Endpoints
 
-- Use TypeScript for new code
-- Follow semantic commit messages
-- Update documentation for new features
-- Add examples for new functionality
-- Test your changes thoroughly
+| Endpoint | Method | Beschrijving |
+|----------|--------|--------------|
```

markdownlint-cli2 (0.20.0)

Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)